### PR TITLE
fix: handle node and children recursively

### PR DESCRIPTION
**Pull request by JounQin**

### CODE REVIEW

File: test/__snapshots__/fixtures.test.ts.snap [hunk 1]
```diff
@@ -206,186 +206,7 @@ Array [
...
-Array [
-  Object {
-    "column": 6,
-    "endColumn": 6,
-    "endLine": 74,
-    "line": 48,
-    "message": "Expected an assignment or function call and instead saw an expression.",
-    "messageId": "unusedExpression",
-    "nodeType": "ExpressionStatement",
-    "ruleId": "no-unused-expressions",
```
Comment (JounQin) [inline test/__snapshots__/fixtures.test.ts.snap:219]: All `no-unused-expressions` issues are gone now!

File: test/__snapshots__/fixtures.test.ts.snap [hunk 2]
```diff
@@ -209,58 +209,177 @@ Array [
...
+            <aui-card> Content 3</aui-card>
+          </aui-tab>
+        </aui-tab-group>
+      ",
+    },
+    "line": 49,
+    "message": "Templates should be properly indented.",
+    "messageId": "template-indent",
+    "nodeType": "TemplateLiteral",
+    "ruleId": "unicorn/template-indent",
```
Comment (JounQin) [inline test/__snapshots__/fixtures.test.ts.snap:247]: Another problem next to be resolved.

Comment (JounQin) [reply test/__snapshots__/fixtures.test.ts.snap:247]: https://github.com/sindresorhus/eslint-plugin-unicorn/issues/1815#issuecomment-1126881651

**Review by wooorm — APPROVED** [review]

**Comment by JounQin** [comment]
@wooorm A new release please.

**Comment by wooorm** [comment]
forgot `yarn build` again 😬

**Comment by wooorm** [comment]
How can I force a publish? 🤔

**Comment by JounQin** [comment]
@wooorm 

https://github.com/lerna/lerna/tree/main/commands/version#--force-publish

**Comment by wooorm** [comment]
It worked, released!

**Pull request merged**